tests/process/spawn.luau

```diff
@@ -138,3 +138,16 @@ assert(
 	echoResult.stdout == (echoMessage .. "\n"), -- Note that echo adds a newline
 	"Inheriting stdio did not return proper output"
 )
+
+-- Passing stdin strings should work
+
+local stdinChild = process.spawn("xargs", {
+	"echo",
+}, {
+	stdin = echoMessage,
+})
+
+assert(
+	stdinChild.stdout == (echoMessage .. "\n"), -- Note that echo adds a newline
+	"Stdin passing did not return proper output"
+)
```

Comment:
Can we use a powershell equivalent when running under Windows for this? I elaborated some more on why specifically to use powershell in tests going forward in #111

Reply:
Possibly could use `powershell` and stdin to echo with the echoMessage for windows.

Example with #111 
```lua
local stdin = if isWindows then `echo "{echoMessage}"\n` else echoMessage
local stdinChild = process.spawn(
	if isWindows then "powershell" else "xargs", 
	not isWindows and {"echo"} or nil, {
	stdin = stdin,
})

local stdOut = stdinChild.stdout
if isWindows then
	-- Windows include the path name and execution
	-- PS <Drive>:\path\to> echo...
	local _, stdinEnd = string.find(stdOut, stdin, 1, true);
	assert(
		stdinEnd,
		"Stdin not found in the output"
	)
	stdinEnd += 1
	stdOut = stdOut:sub(stdinEnd, stdinEnd + #echoMessage + 1)
end

assert(
	stdOut == (echoMessage .. trailingAddition), -- Note that echo adds a newline
	"Stdin passing did not return proper output"
)
```

Reply:
Would be above test example be applicable in this case? I'm not really familiar with powershell scripting (*as I'm a GNU-er!* 😎).

Reply:
Looks like `echo` on windows reads for stdin, so something like `xargs` would not be needed altogether.

Reply:
Addressed in https://github.com/filiptibell/lune/pull/106/commits/7dd9223157a1553e5fa6adbc19cac4c86362e3a2 & https://github.com/filiptibell/lune/pull/106/commits/7dd9223157a1553e5fa6adbc19cac4c86362e3a2.